packages/rapidpro/src/Adaptor.js

```diff
@@ -1,12 +1,9 @@
 import {
   execute as commonExecute,
   composeNextState,
-  http, // Important: this is the OLD axios-based http.
 } from '@openfn/language-common';
 import { expandReferences } from '@openfn/language-common/util';
-
-const { axios } = http;
```

Comment:
Just to be clear - this is a breaking change. Removing an exported function is always a breaking change.